modules/rosa-accessing-your-cluster-quick.adoc

```diff
@@ -30,22 +30,24 @@ $ rosa create admin --cluster=<cluster_name>
 W: It is recommended to add an identity provider to login to this cluster. See 'rosa create idp --help' for more information.
 I: Admin account has been added to cluster 'cluster_name'. It may take up to a minute for the account to become active.
 I: To login, run the following command:
-oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \
+oc login https://api.cluster-name.t6k4.i1.oragnization.org:6443 \ <1>
 --username cluster-admin \
 --password FWGYL-2mkJI-3ZTTZ-rINns
 ----
+<1> For a {hcp-title-first} cluster, the port number should be `443`.
 
 . Enter the `oc login` command, username, and password from the output of the previous command:
 
 +
 .Example output
 [source,terminal]
 ----
-$ oc login https://api.cluster_name.t6k4.i1.oragnization.org:6443 \
+$ oc login https://api.cluster_name.t6k4.i1.oragnization.org:6443 \ <1>
 >  --username cluster-admin \
 >  --password FWGYL-2mkJI-3ZTTZ-rINns
 Login successful.                                                                                                                                                                                                                                                       You have access to 77 projects, the list has been suppressed. You can list all projects with ' projects'
```

Comment:
This is out of scope for your PR, but is the line that starts with "You have access to 77 projects..." supposed to appear on a new line in the terminal? In the rendered doc, it appears on the same line as "Login successful" but very far to the right, causing this code block to have a horizontal scroll because of its width. 
```suggestion
Login successful.                                                                                                                                                                                                                                                       
You have access to 77 projects, the list has been suppressed. You can list all projects with ' projects'
```
Up to you whether you want to fix this, if applicable, since this isn't something you changed.

Reply:
Created a Jira: https://issues.redhat.com/browse/OSDOCS-9968 to implement out of scope comments for this PR. Thank you!